Source/WebCore/rendering/mathml/RenderMathMLBlockInlines.h

```diff
@@ -52,12 +52,10 @@ inline LayoutUnit RenderMathMLBlock::mirrorIfNeeded(LayoutUnit horizontalOffset,
     return mirrorIfNeeded(horizontalOffset, child.logicalWidth());
 }
 
+// https://w3c.github.io/mathml-core/#dfn-default-rule-thickness
 inline LayoutUnit RenderMathMLBlock::ruleThicknessFallback() const
 {
-    // This function returns a value for the default rule thickness (TeX's \xi_8) to be used as a fallback when we lack a MATH table.
-    // This arbitrary value of 0.05em was used in early WebKit MathML implementations for the thickness of the fraction bars.
-    // Note that Gecko has a slower but more accurate version that measures the thickness of U+00AF MACRON to be more accurate and otherwise fallback to some arbitrary value.
-    return LayoutUnit(0.05f * style().fontCascade().size());
+    return LayoutUnit(style().metricsOfPrimaryFont().underlineThickness().value_or(0.0f));
```

Comment:
Would using `checkedStyle()` be enough to fix the safer CPP warning?

Reply:
@fred-wang - Thanks for help. It seems `checkedStyle()` worked. :-)